server/app/models/ckb_sync/persist.rb

```diff
@@ -225,7 +226,8 @@ def build_cell_input(ckb_transaction, input)
         ckb_transaction.cell_inputs.build(
           previous_output: input["previous_output"],
           since: input["since"],
-          args: input["args"]
+          args: input["args"],
+          from_cell_base: input["previous_output"]["cell"].present? ? false : true
```

Comment:
`! input["previous_output"]["cell"].present?` maybe more better ?

Reply:
Thank you, I found it more appropriate to use `blank?`.